test/simulation_time/simulation_clock_Test.cpp

```diff
@@ -0,0 +1,47 @@
+#include "gtest/gtest.h"
+
+#include <simulation_time/simulation_clock.hpp>
+
+using clock_type = ngen::simulation_clock;
+
+// 1704067200 -> 01-01-2024 00:00:00 UTC
+constexpr std::time_t epoch_unix = 1704067200L;
+const auto epoch_calendar = clock_type::calendar_clock::from_time_t(epoch_unix);
+
+TEST(SimulationClockTest, TestConfiguration)
+{
+    ASSERT_FALSE(clock_type::has_epoch());
+    ASSERT_NO_THROW(clock_type::set_epoch(epoch_calendar));
+    ASSERT_TRUE(clock_type::has_epoch());
+    ASSERT_THROW(clock_type::set_epoch(epoch_calendar), std::logic_error);
+    EXPECT_EQ(clock_type::epoch(), epoch_calendar);
+}
+
+TEST(SimulationClockTest, TestConversion)
```

Comment:
It looks like this test assumes `TestConfiguration` has already run before it, setting the `simulation_clock`'s epoch. With tests run in parallel, is that dependency still honored? I don't know how concurrent gtest can get, and whether our usage of it otherwise allows for that concurrency.

In general, it's not great to have inter-test dependencies. In this case, the easy solution may be simply to merge `TestConfiguration` and `TestConversion` into a single test. Most of `TestTicking` is fine, until the very last `EXPECT_EQ` that references `epoch_calendar. So, maybe merge them all into a single sequence?